go.mod

```diff
@@ -45,3 +45,8 @@ require (
 	google.golang.org/grpc v1.46.2
 	gopkg.in/yaml.v3 v3.0.1
 )
+// `runc` is referenced by `github.com/adlio/schema`.
+// This is a temporary fix for a security vulnerability of `runc`.
+// So, remove this `replace` when `github.com/adlio/schema` releases a version that references `runc 1.1.2`.
+// For details, see here https://github.com/line/ostracon/security/dependabot/3`.
```

Comment:
https://github.com/line/ostracon/pull/429/files#r891885989

Please add a public URL with keeping this URL.

Reply:
I will fix.